README.md

```diff
@@ -10,15 +10,37 @@ Sporkfish is a Python-based chess engine. Chess programming techniques, although
 
 ### Set up
```

Comment:
If you don't mind, shall we just move Setup into its own section now? Its getting heavy now, we have sub-sub-sub-sub sections now ;)